prov/rxm/src/rxm_ep.c

```diff
@@ -149,6 +149,8 @@ static void rxm_ep_txrx_res_close(struct rxm_ep *rxm_ep)
 	while(!slist_empty(&rxm_ep->rx_buf_list)) {
 		entry = slist_remove_head(&rxm_ep->rx_buf_list);
 		rx_buf = container_of(entry, struct rxm_rx_buf, entry);
+		/* Cancel pre-posted context and relese it */
+		(void)fi_cancel(&rx_buf->ep->srx_ctx->fid, rx_buf);
```

Comment:
fi_cancel returns int value as a resut, this is needed just to suppress warning if lib is built with -Wno-unused-value gcc flag, for example.